package.json

```diff
@@ -33,31 +33,29 @@
     "version": "npm run build && git add -A dist"
   },
   "dependencies": {
-    "append-query": "^1.1.0",
-    "core-js": "^2.4.1",
-    "es6-error": "^3.2.0",
-    "es6-promise": "^4.0.4",
-    "fast-memory-cache": "^2.0.4",
-    "hellojs": "^1.13.1",
-    "local-storage": "^1.4.2",
-    "lodash": "^4.16.0",
-    "loglevel": "^1.4.1",
-    "promise-queue": "^2.2.3",
-    "qs": "^6.2.0",
+    "append-query": "1.1.0",
```

Comment:
We should be installing specific versions. For now we haven't had an issue but we are risking the chance that things could not work if a new version is released for a lib we are using.